package.json

```diff
@@ -25,14 +31,20 @@
     "typeface-arimo": "^0.0.22"
   },
   "devDependencies": {
-    "react-scripts": "^3.1.1"
+    "react-scripts": "^3.4.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "node scripts/build-non-split.js && npm run build:clean",
     "build:clean": "cd build/static/js/ && cp *.js ./main.js && cp *.map ./main.js.map && cp ../../service-worker.js .",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "format": "npx prettier --write \"src/**/*.js\""
+  },
+  "prettier": {
```

Comment:
This is a nice addition to the project. I know that our most recent change with the `"printWidth": 100` came after this PR, but maybe it could be added in now as well? Since there's already a lot changed because of prettier formatting anyway...

Reply:
fair point. i added it in #212 but i can add it here